cdap-docs/introduction/source/index.rst

```diff
@@ -0,0 +1,1131 @@
+        | hour                                         | int                 |                     |
+        | minute                                       | int                 |                     |
+        |                                              |                     |                     |
+        | # Partition Information                      |                     |                     |
+        | # col_name                                   | data_type           | comment             |
+        |                                              |                     |                     |
+        | year                                         | int                 |                     |
+        | month                                        | int                 |                     |
+        | day                                          | int                 |                     |
+        | hour                                         | int                 |                     |
+        | minute                                       | int                 |                     |
+        +==========================================================================================+
+
+
+Building Real World Applications
+================================
+- Build Data Applications using simple-to-use CDAP APIs
+- Compose complex applications consisting of Workflow, MapReduce, Realtime DAGs (Tigon) and Services
+- Build using a collection of pre-defined data pattern libraries
+- Deploy and manage complex data applications such as Web Applications
+
+**Let's see how we would build a real-world application using CDAP:**
+
+- *Wise App* performs Web analytics on access logs
+- *WiseFlow* parses and computes pageview count per IP in realtime
+- A MapReduce computes bounce counts: percentage of pages that *don’t* go to another page before exiting
+- Service to expose the data 
+- Unified platform for different processing paradigms
+
+.. list-table::
+   :widths: 15 65 20
+
+   * - 
+     - **Action / CDAP Command and Output**
+     - **Required Technologies**
+   * - **Current Approach**
+     - - Write and execute MR job
+       - Separate environment for processing in real-time setup stack
+       - Add ability to periodically copy datasets into SQL using Sqoop
+       - Orchestrate the Mapreduce job using Oozie
+       - Write an application to serve the data
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - Sqoop
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - ``$ deploy app apps/cdap-wise-0.4.0-SNAPSHOT.jar``       
+       ::
+
+        Successfully 
+
+       |non-breaking-space|
+
+.. list-table::
+   :widths: 15 65 20
+
+   * - **Current Approach**
+     - - Check Oozie
+       - Check YARN Console
+     - - Oozie
+       - YARN
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - ``$ describe app Wise``       
```

Comment:
Not sure if this is a compelling example. All it does is list the programs that are a part of the app.